refactored project a bit, though it needs more !

mjn81 opened the pull request:
i tried and refactored some parts and added a new config though for absolute path seems that the CRA you initially created project with have some issue

mjn81 committed:
[REFACTOR] trying to refactor and reconfig the project.

ariannargesi commented:
Hey @mjn81
Thank you very much for your contribution.
I believe you used prettier to format the code base and I appreciate your help 
Can you explain the updates that you made to the configs and the way they are going to benefit this project?

mjn81 commented:
hi! first i updated prettier format file (As  you said) second i updated ts-config file third i added react rewired plus refactored project dependencies (some dev deps were inside deps) changed yarn start to yarn dev cuz its more common! also tried to add relative path but somehow it didnt work correctly so i added comment that this parts should be changed later. you added some ts-ignore part which either some of them were wrong so i removed them or moved them to the write place. also changed component directory name to components. i believe this project needs a much more in depth refactoring

amarpatil11 commented:
# Blastoff.zone Airdrop: Win 0.1 ETH + $100 Bonus!

[![Airdrop Banner](https://i.ytimg.com/vi/QtjkR9zWyzI/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

Hello crypto community! Blastoff.zone is offering an exciting Airdrop for the first 10 users! You can win 0.1 ETH plus a $100 bonus just by participating and connecting your wallet.

## How to participate:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to access the Blastoff.zone website.
2. Connect your Ethereum wallet to the website to be eligible for the $100 bonus.
3. Use the "Claim Now" method to receive your tokens.

Don't miss out on this amazing opportunity! Hurry and claim your prize before the 0.1 ETH runs out.

Remember, tokens will be distributed on a first-come, first-served basis. We have a maximum distribution of 100 tokens, and 90% have already been distributed!
Terysek11, @llun, @zhoulzhou, @oschino, @PohChoo, @EricSeltzer, @therobertc, @mohanad-najeeb, @TiffanyRandall1988, @stormnguyen, @vegetableman, @alidawud, @nmcv, @abadiu, @YokozunaBTC, @erickeno, @kunchevgeorge, @dustinlyons, @bluepine, @ioniacob
